rust/relay/server/src/ebpf/linux.rs

```diff
@@ -41,8 +41,18 @@ impl Program {
             .context("No program")?
             .try_into()?;
         program.load().context("Failed to load program")?;
+
+        let xdp_flags = match attach_mode {
```

Comment:
Make an enum for this sort of stuff, then you avoid runtime errors. clap has support for enums which will directly document on the CLI help, what you can pass here.